doc/content/getting-started/api/_index.md

```diff
@@ -0,0 +1,205 @@
+      "ids": {
+        "device_id": "newdev1",
+        "dev_eui": "XXXXXXXXXXXXXXXX",
+        "join_eui": "XXXXXXXXXXXXXXXX"
+      },
+      "lorawan_phy_version": "1.0.2-b",
+      "frequency_plan_id": "EU_863_870_TTN"
+    },
+    "field_mask": {
+      "paths": [
+        "supports_join",
+        "lorawan_version",
+        "ids.device_id",
+        "ids.dev_eui",
+        "ids.join_eui",
+        "lorawan_phy_version",
+        "frequency_plan_id"
+      ]
+    }
+  }' \
+  'https://thethings.example.com/api/v3/ns/applications/app1/devices/newdev1'
+```
+
+Register the `DevEUI` and `JoinEUI` in the Application Server:
+
+```bash
+$ curl --location \
+  --header 'Authorization: Bearer NNSXS.XXXXXXXXX' --header 'Content-Type: application/json' \
+  --header 'User-Agent: my-integration / my-integration-version' \
+  --request PUT \
+  --data-raw '{
+    "end_device": {
+      "ids": {
+        "device_id": "newdev1",
+        "dev_eui": "XXXXXXXXXXXXXXXX",
+        "join_eui": "XXXXXXXXXXXXXXXX"
+      }
+    },
+    "field_mask": {
+      "paths": [
+        "ids.device_id",
+        "ids.dev_eui",
+        "ids.join_eui"
+      ]
+    }
+  }' \
+  'https://thethings.example.com/api/v3/as/applications/app1/devices/newdev1'
+```
+
+Finally, register the `DevEUI`, `JoinEUI`, cluster addresses, and keys in the Join Server:
+
+```bash
+$ curl --location \
+  --header 'Authorization: Bearer NNSXS.XXXXXXXXX' --header 'Content-Type: application/json' \
+  --header 'User-Agent: my-integration / my-integration-version' \
+  --request PUT \
+  --data-raw '{
+    "end_device": {
+      "ids": {
+        "device_id": "newdev1",
+        "dev_eui": "XXXXXXXXXXXXXXXX",
+        "join_eui": "XXXXXXXXXXXXXXXX"
+      },
+      "network_server_address": "thethings.example.com",
+      "application_server_address": "thethings.example.com",
+      "root_keys": {
+        "app_key": {
+          "key": "XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX"
+        }
+      }
+    },
+    "field_mask": {
+      "paths": [
+        "network_server_address",
+        "application_server_address",
+        "ids.device_id",
+        "ids.dev_eui",
+        "ids.join_eui",
+        "root_keys.app_key.key"
+      ]
+    }
+  }' \
+  'https://thethings.example.com/api/v3/js/applications/app1/devices/newdev1'
+```
+
+## Best Practices
+
+### Send a User-Agent Header
+
+Set a User-Agent header containing your integration name and version. That way, a network operator can target specific integrations and versions in their firewall rules in case some integration misbehaves, instead of blocking all versions of curl or your integration completely.
+
+### Look for X-Ratelimit-* Response Headers
+
+{{% tts %}} sends responses containing information about how many requests your integration has made and how many are remaining. See [here](https://auth0.com/docs/policies/rate-limit-policy#review-http-response-headers) for more information about how to act on this information.
```

Comment:
It feels a bit weird to refer to the API documentation of Auth0 for this. 

If we want to provide background, we could point to the internet-draft (https://tools.ietf.org/id/draft-polli-ratelimit-headers-03.html) and explain that we use the `X-` prefix until it becomes an RFC.